### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by nschonni**
Took another pass with cSpell. Tried to avoide the namespace ones that are more likely product bugs

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/architecture/cloud-native/relational-vs-nosql-data.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/architecture/cloud-native/relational-vs-nosql-data.md) | [Relational vs. NoSQL data](https://review.learn.microsoft.com/en-us/dotnet/architecture/cloud-native/relational-vs-nosql-data?branch=pr-en-us-34881) |
| [docs/architecture/maui/dependency-injection.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/architecture/maui/dependency-injection.md) | [Dependency injection](https://review.learn.microsoft.com/en-us/dotnet/architecture/maui/dependency-injection?branch=pr-en-us-34881) |
| [docs/core/diagnostics/diagnosticsource-diagnosticlistener.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/core/diagnostics/diagnosticsource-diagnosticlistener.md) | [DiagnosticSource and DiagnosticListener](https://review.learn.microsoft.com/en-us/dotnet/core/diagnostics/diagnosticsource-diagnosticlistener?branch=pr-en-us-34881) |
| [docs/core/install/macos.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/core/install/macos.md) | [Install .NET on macOS](https://review.learn.microsoft.com/en-us/dotnet/core/install/macos?branch=pr-en-us-34881) |
| [docs/core/install/windows.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/core/install/windows.md) | [[.NET 6](#tab/net60)](https://review.learn.microsoft.com/en-us/dotnet/core/install/windows?branch=pr-en-us-34881) |
| [docs/csharp/advanced-topics/performance/ref-tutorial.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/csharp/advanced-topics/performance/ref-tutorial.md) | [Tutorial: Reduce memory allocations with `ref` safety](https://review.learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/performance/ref-tutorial?branch=pr-en-us-34881) |
| [docs/csharp/whats-new/tutorials/interpolated-string-handler.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/csharp/whats-new/tutorials/interpolated-string-handler.md) | [Tutorial: Write a custom string interpolation handler](https://review.learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/interpolated-string-handler?branch=pr-en-us-34881) |
| [docs/framework/additional-apis/pos-for-net/linedisplay-capabilities.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/framework/additional-apis/pos-for-net/linedisplay-capabilities.md) | [LineDisplay Capabilities (POS for .NET v1.14 SDK Documentation)](https://review.learn.microsoft.com/en-us/dotnet/framework/additional-apis/pos-for-net/linedisplay-capabilities?branch=pr-en-us-34881) |
| [docs/framework/data/adonet/common-schema-collections.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/framework/data/adonet/common-schema-collections.md) | [Common Schema Collections](https://review.learn.microsoft.com/en-us/dotnet/framework/data/adonet/common-schema-collections?branch=pr-en-us-34881) |
| [docs/framework/data/adonet/oracle-lobs.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/framework/data/adonet/oracle-lobs.md) | [Oracle LOBs](https://review.learn.microsoft.com/en-us/dotnet/framework/data/adonet/oracle-lobs?branch=pr-en-us-34881) |
| [docs/framework/data/adonet/sql/sqlcommand-execution-with-a-sqlnotificationrequest.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/framework/data/adonet/sql/sqlcommand-execution-with-a-sqlnotificationrequest.md) | [SqlCommand Execution with a SqlNotificationRequest](https://review.learn.microsoft.com/en-us/dotnet/framework/data/adonet/sql/sqlcommand-execution-with-a-sqlnotificationrequest?branch=pr-en-us-34881) |
| [docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md) | [How to: Determine which .NET Framework versions are installed](https://review.learn.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed?branch=pr-en-us-34881) |
| [docs/framework/unmanaged-api/debugging/icordebug-setunmanagedhandler-method.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/framework/unmanaged-api/debugging/icordebug-setunmanagedhandler-method.md) | [ICorDebug::SetUnmanagedHandler Method](https://review.learn.microsoft.com/en-us/dotnet/framework/unmanaged-api/debugging/icordebug-setunmanagedhandler-method?branch=pr-en-us-34881) |
| [docs/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions.md) | [IdentityModel Exceptions](https://review.learn.microsoft.com/en-us/dotnet/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions?branch=pr-en-us-34881) |
| [docs/fundamentals/networking/quic/quic-overview.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/fundamentals/networking/quic/quic-overview.md) | [QUIC protocol](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/networking/quic/quic-overview?branch=pr-en-us-34881) |
| [docs/machine-learning/how-to-guides/prepare-data-ml-net.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/machine-learning/how-to-guides/prepare-data-ml-net.md) | [Prepare data for building a model](https://review.learn.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/prepare-data-ml-net?branch=pr-en-us-34881) |
| [docs/machine-learning/how-to-guides/serve-model-web-api-ml-net.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/machine-learning/how-to-guides/serve-model-web-api-ml-net.md) | [Deploy a model in an ASP.NET Core Web API](https://review.learn.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/serve-model-web-api-ml-net?branch=pr-en-us-34881) |
| [docs/orleans/host/configuration-guide/serialization-configuration.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/orleans/host/configuration-guide/serialization-configuration.md) | [Serialization configuration in Orleans](https://review.learn.microsoft.com/en-us/dotnet/orleans/host/configuration-guide/serialization-configuration?branch=pr-en-us-34881) |
| [docs/orleans/host/configuration-guide/server-configuration.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/orleans/host/configuration-guide/server-configuration.md) | [Server configuration](https://review.learn.microsoft.com/en-us/dotnet/orleans/host/configuration-guide/server-configuration?branch=pr-en-us-34881) |
| [docs/standard/data/xml/user-defined-functions-and-variables.md](https://github.com/dotnet/docs/blob/03a4db7fb221bfefd13df5a59f83ce2793a212f9/docs/standard/data/xml/user-defined-functions-and-variables.md) | [User Defined Functions and Variables](https://review.learn.microsoft.com/en-us/dotnet/standard/data/xml/user-defined-functions-and-variables?branch=pr-en-us-34881) |

</details>

### CODE REVIEW

File: docs/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ This topic lists all exceptions generated by IdentityModel.
 |TraceCodeSecurityPendingServerSessionRemoved|A pending security session was made active by the server.|  
 |TraceCodeImportSecurityChannelBindingExit|Finished Security ImportChannelBinding.|  
 |X509CertStoreLocationNotValid|The StoreLocation must be either LocalMachine or CurrentUser.|  
-|SettingdMayBeModifiedOnlyWhenTheWriterIsInStartState|The writer settings may be modified only when the writer is in the Start state.|  
+|SettingsMayBeModifiedOnlyWhenTheWriterIsInStartState|The writer settings may be modified only when the writer is in the Start state.|
```
Comment (nschonni) [inline docs/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions.md:242]: This one might be a product bug, but I didn't try to trace down to the code

Comment (BillWagner) [reply docs/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions.md:242]: ping @HongGit for this change. "Settings" vs. "Settingd" in the resource string.

I can't find any instance of either spelling in our docs.

The rest of the PR looks fine, but I'm not sure which is correct here.

Comment (gewarren) [reply docs/framework/wcf/diagnostics/exceptions-reference/identitymodel-exceptions.md:242]: I don't think the resource name is that important anyway, so can merge as-is.

**Comment by gewarren** [comment]
One of the snippets errors will go away once #34894 is merged.

**Comment by nschonni** [comment]
Sounds good. Just ping me if this needs a rebase after that lands, as it looks like the merge conflict job got disabled

**Comment by BillWagner** [comment]
Hi @nschonni 
#34894 has been merged. Both the conflicting files have been deleted in that PR.

**Review by gewarren — APPROVED** [review]

**Pull request merged**